### PR TITLE
Add The Identity Hub

### PR DESCRIPTION
**Pull request by TheIdentityHub**

### CODE REVIEW

**Review by stephengroat — DISMISSED** [review]

File: _data/identity.yml [hunk 1]
```diff
@@ -158,6 +158,15 @@ websites:
           text: "2FA is only available on paid plans."
       doc: https://www.securesafe.com/en/faq/general-questions/how-can-i-activate-2-factor-authentication/
 
+    - name: The Identity Hub
+      url: https://www.theidentityhub.com
+      img: theidentityhub.png
+      tfa: Yes
+      sms: Yes
```
Comment (Carlgo11) [inline _data/identity.yml:165]: I don't see anything on the documentation page you provided about SMS authentication. Are you sure you support it?

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/identity.yml [hunk 2]
```diff
@@ -158,6 +158,15 @@ websites:
           text: "2FA is only available on paid plans."
       doc: https://www.securesafe.com/en/faq/general-questions/how-can-i-activate-2-factor-authentication/
 
+    - name: The Identity Hub
+      url: https://www.theidentityhub.com
```
Comment (Carlgo11) [inline _data/identity.yml]: When I go to `https://www.theidentityhub.com` I get redirected to `https://shop.theidentityhub.com/`. Is that just me or is indeed your homepage `https://shop.theidentityhub.com/`?

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/identity.yml (hunk 1 above)
Comment (TheIdentityHub) [reply _data/identity.yml:165]: Only one link to documentation was possible. Here https://www.theidentityhub.com/hub/Documentation/#ConfigureSmsProvider is the explaination on how to configure a SMS provider that will allow SMS authentication.

File: _data/identity.yml (hunk 2 above)
Comment (TheIdentityHub) [reply _data/identity.yml]: The home page is https://www.theidentityhub.com, the redirect to https://shop.theidentityhub.com has been introduced to separate the commercial front-end from the main site. The documentation site and the admin site are in the https://www.theidentityhub.com domain (https://www.theidentityhub.com/hub/Documentation and https://www.theidentityhub.com/hub/Authenticate).

Comment (TheIdentityHub) [reply _data/identity.yml]: Hello,

Any news on this? Do we need to make some changes for this to be approved?

Regards,
The Identity Hub

Comment (Carlgo11) [reply _data/identity.yml]: Well, as the url value you have set redirects to the shop subdomain I'm afraid the site can't be listed. So you'll either have to change the url value to `https://shop.theidentityhub.com/` or stop the redirection.

Comment (TheIdentityHub) [reply _data/identity.yml]: I changed the url to shop.theidentityhub.com and added a new commit.

File: _data/identity.yml (hunk 1 above)
Comment (TheIdentityHub) [reply _data/identity.yml:165]: We have updated our documentation: https://www.theidentityhub.com/hub/Documentation/#TwoFactorAuthentication

Is this sufficient?

Regards
Hannes

**Comment by TheIdentityHub** [comment]
Hello,

is there anything else we need to do before this pull request can be merged?

regards
The Identity Hub

**Comment by brycx** [comment]
@TheIdentityHub thank you for the PR, but it doesn't meet requirement 4. in the contributions guidelines "Alexa top 200k" and can therefor not be included at this point in time.

**Pull request closed without merging**